dts/bindings/interrupt-controller/litex,vexriscv-intc0.yaml

```diff
@@ -3,7 +3,7 @@
 
 description: LiteX VexRiscV interrupt controller
 
-compatible: "vexriscv-intc0"
+compatible: "litex,vexriscv-intc0"
```

Comment:
Need to add a note to the 3.7 migration guide about the changed compatible

Reply:
done